Gemfile

```diff
@@ -37,8 +37,8 @@ gem 'bootsnap', '>= 1.4.2', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
-  gem 'factory_bot_rails'
-  gem 'faker'
+  gem 'factory_bot_rails', '~> 6.2.0'
+  gem 'faker', '~> 2'
```

Comment:
i am not sure if faker should be here. In other projects i saw faker in general scope.
CC

Reply:
No problem. It's normal place.